horizon/Chart.yaml

```diff
@@ -17,13 +17,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.0.0
+version: 1.0.1
```

Comment:
We shouldn't increment the chart versions unless we are ready to publish all changes in the main branch.